libs/core/langchain_core/tracers/context.py

```diff
@@ -31,21 +29,14 @@
     from langchain_core.callbacks.base import BaseCallbackHandler, Callbacks
     from langchain_core.callbacks.manager import AsyncCallbackManager, CallbackManager
 
-# Deprecated as of 0.1.0, will be removed in 0.2.0.
-tracing_callback_var: ContextVar[Optional[LangChainTracerV1]] = ContextVar(  # noqa: E501
-    "tracing_callback", default=None
-)
-
-tracing_v2_callback_var: ContextVar[Optional[LangChainTracer]] = ContextVar(  # noqa: E501
+tracing_v2_callback_var: ContextVar[Optional[LangChainTracer]] = ContextVar(
     "tracing_callback_v2", default=None
-)
-run_collector_var: ContextVar[Optional[RunCollectorCallbackHandler]] = ContextVar(  # noqa: E501
+)  # noqa: E501
+run_collector_var: ContextVar[Optional[RunCollectorCallbackHandler]] = ContextVar(
     "run_collector", default=None
-)
+)  # noqa: E501
 
 
-@contextmanager
```

Comment:
is this breaking?

Reply:
The whole thing is breaking, but agreed not having `@contextmanager` will make the error message more esoteric (can't do `with tracing_enabled()`, instead of the intentional runtime error.

Will add back

Reply:
Done